spec/govuk_design_system_formbuilder/builder/label_spec.rb

```diff
@@ -0,0 +1,75 @@
+describe GOVUKDesignSystemFormBuilder::FormBuilder do
+  include_context 'setup builder'
+
+  describe '#govuk_password_field' do
+    let(:custom_label_text) { "What's your name?" }
+    let(:method) { :govuk_label }
+    let(:attribute) { :name }
+    let(:args) { [method, attribute] }
+    let(:kwargs) { { text: custom_label_text } }
+
+    subject { builder.send(*args, **kwargs) }
+
+    specify 'renders a label with the right custom text' do
+      expect(subject).to have_tag('label', text: custom_label_text, with: { class: 'govuk-label' })
```

Comment:
Worth adding a test for the `for` attribute too, given that this will be needed to link the label to the input?

Reply:
Missed this, but yeah it is. Will add it.